src/components/layout/layout.jsx

```diff
@@ -0,0 +1,32 @@
+import Head from "next/head";
+import Footer from "../footer/footer";
+import Header from "../header/header";
+import layoutStyle from "./layout.module.css";
+
+const Layout = ({main, children}) => {
+
+  return (
+    <>
+      <Head>
+        <title>Create Next App</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <div className={layoutStyle.layout}>
+        {!main ? <Header className="container" /> : ``}
```

Comment:
Лучше возвращать `null`, потому что часто мы рендерим что-то или null. Хотя может он в этом кейсе превратится в строку. Но кажется, что не должен

```suggestion
        {!main ? <Header className="container" /> : null}
```

Reply:
Исправлю на null